deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2

```diff
@@ -32,7 +32,7 @@ postgres:
   port: 5432
 
 core:
-  logging_level: debug
+  logging_level: info
```

Comment:
@aliaksandr-d was that the only reason of airflow pods fails? If we will need the debug level again we will face the same problem?

Reply:
logging_level was decreased to INFO to reduce Airflow load.